windows/installer/MozillaVPN_prod.wxs

```diff
@@ -74,7 +74,7 @@
 		-->
     <ComponentGroup Id="MozillaVPNComponents">
       <Component Directory="MozillaVPNFolder" Id="MozillaVPNExecutable" Guid="EF353A73-A10B-4A2D-87AC-4389BBD9313B">
-        <File Id="MozillaVPNExecutable" Source="../../MozillaVPN.exe" KeyPath="yes">
+        <File Id="MozillaVPNExecutable" Source="../../Mozilla VPN.exe" KeyPath="yes">
```

Comment:
I don't know how the installer really works under the hood. 
Would this mean on 2.8->2.9 upgrade, the user would have 2 .exe's or does the installer keep a receipt and checks that Id="MozillaVPNExecutable" did now points to a diffrent thing (and do cleanup) ?

Reply:
no, the installer removes the previous files. Nothing is left of the previous installation files.